code/api/src/migrations/20210325155429-5-userproducts.js.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable("userProducts", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: "users",
+          key: "id"
+        },
+        allowNull: false
+      },
+      crateId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: "crates",
+          key: "id"
+        },
+      },
+      kept: {
+        type: Sequelize.BOOLEAN
+      },
+      deliveryDate: {
+        type: Sequelize.DATE
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
```

Comment:
I understand what the `up` creating the table does, does this `down/droptable` just delete the table that has the key/title of `UserProducts`?

Reply:
Are up/down another way to say request/reponse?

Reply:
`up/down` are in regards to migrations and rollbacks. If we run a rollback on the migration it will drop/delete the table in this instance. Similar concept to github commit rollbacks